tools/cert_create/Makefile

```diff
@@ -47,6 +55,9 @@ else
 	Q :=
 endif
 
+$(eval $(call add_define,USE_TBBR_DEFS))
+CFLAGS += ${DEFINES}
+
```

Comment:
I could not understand what you mean.

I think my code passes `USE_TBBR_DEFS` to C files unconditionally.
What is your intention of moving it to line 24?